src/tile/tile.scss

```diff
@@ -269,7 +270,8 @@
     }
 
     .iui-picture {
-      transform: translateZ(0) scale(1.1);
+      width: calc(100% + #{$iui-xl});
```

Comment:
I got curious about this theory so I put `width: 300px; height: 300px` on tile itself, and I was still getting blurry text.

I also tried round the dimensions of the thumbnail but no luck.